generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -492,6 +493,7 @@ public class <%= asEntity(entityClass) %> implements Serializable {
         const otherEntityNamePlural = relationships[idx].otherEntityNamePlural;
         const otherEntityRelationshipNameCapitalized = relationships[idx].otherEntityRelationshipNameCapitalized;
         const otherEntityRelationshipNameCapitalizedPlural = relationships[idx].otherEntityRelationshipNameCapitalizedPlural;
+        const isOtherEntityFromUserManagement = asEntity(otherEntityNameCapitalized) === 'User' || asEntity(otherEntityNameCapitalized) === 'Authority';
```

Comment:
`isOtherEntityFromUserManagement` -> `userManagementEntity`
I initially thought it was a function (like isBlue) but it's a variable, no need to use a verb here